CNI/romana

```diff
@@ -154,9 +155,9 @@ set_up_pod () {
 	# Asking romana ipam for an IP address, based on `tenant` label.
 	log "--- PRE-IPAM ---"
 	[[ $SEGMENT ]] || SEGMENT=default
-	IP=$(ipam_allocate_ip $TENANT $SEGMENT $NODE)
+	IP=$(ipam_allocate_ip $NAMESPACE $SEGMENT $NODE)
```

Comment:
I guess I would have kept the TENANT variable and just set TENANT=$NAMSPACE somewhere up ahead. That way, we can see immediately what that variable is really used for here. Also, if ever the k8s community decides to abandon the NS==Tenant idea then we just would have had to change one line.